autoarena/store/database.py

```diff
@@ -26,3 +26,7 @@ def get_database_connection(path: Path) -> duckdb.DuckDBPyConnection:
         yield conn
     finally:
         conn.close()
+
+
+def get_available_migrations() -> list[Path]:
+    return sorted(MIGRATION_DIRECTORY.glob("[0-9][0-9][0-9]__*.sql"))
```

Comment:
i think prefixing the timestamp like `YYYYMMDDHHMMSS` for instance `20240502100843` will be sorted naturally and avoid the number of digits limitation

Reply:
If we ever get close to 1000 migrations we'll make a checkpoint release! It's easier to write the next int than it is to handcraft a date string like that.

Reply:
what do you mean a checkpoint release? compact all migrations into a single new sql `1000__checkpoint.sql`?

i think having some scripts to generate the placeholder new sql files with proper naming convention is not that hard

Reply:
A checkpoint release squashes the migrations into a single file and lets you start over from `000`. It'll be an easy extension to add that to this migration system when the time comes (it shouldn't for quite some time though).

I don't want to have to use a script to generate a new migration — just create a new file. Devx is king

Reply:
in my understanding, we are using the file name as id for migrations: https://github.com/kolenaIO/autoarena/pull/44/files#diff-65d61ea14a619e4cc321bbc62ff87b605b822abed4dc873acfa925556c75c7d9R70, starting over from `000` might cause file name duplicating. that's why i suggest to use timestamp as a monotonic prefix to avoid this kind of issues.